fix: Remove duplicated (unclosed) section tag

rediris opened the pull request:
## Description

This PR removes some bad (duplicated) markup in the static tab file. The current behavior leads to an unclosed HTML tag, which can be confirmed by viewing the document source.

JadeyOlivier committed:
feat: Commerce Coordinator step in retirement pipeline

JadeyOlivier committed:
fix: quality

hamzawaleed01 committed:
feat: add PA role

hamzawaleed01 committed:
feat: add PA role

JadeyOlivier committed:
fix: generalize internal services

navinkarkera committed:
fix: delete discarded drafts from meilisearch index

navinkarkera committed:
refactor: parallelize content block update on discard

navinkarkera committed:
refactor: delete all documents on discard

navinkarkera committed:
refactor: delete documents that were never published on discard

navinkarkera committed:
test: delete documents that were never published on discard

MueezKhan246 committed:
feat: Upgrade Python dependency edx-enterprise

added logs for data dog inspection for permission check in handler function.

Commit generated by workflow `openedx/edx-platform/.github/workflows/upgrade-one-python-dependency.yml@refs/heads/master`

JadeyOlivier committed:
fix: tests

JadeyOlivier committed:
fix: Add CI update for tests

JadeyOlivier committed:
fix: dependencies

JadeyOlivier committed:
Revert "fix: dependencies"

This reverts commit 2659ae9fff3ec9a85298cae623097177bd869659.

rediris committed:
fix: Remove duplicated (unclosed) section tag

openedx-webhooks commented:
Thanks for the pull request, @rediris!

### What's next?

*Please work through the following steps to get your changes ready for engineering review:*

#### :radio_button: Get product approval

If you haven't already, [check this list](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3875962884/How+to+submit+an+open+source+contribution+for+Product+Review#Does-my-contribution-require-Product-Review%3F) to see if your contribution needs to go through the product review process.

- If it does, you'll need to submit a product proposal for your contribution, and have it reviewed by the [Product Working Group](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3449028609/Product+Working+Group).
    - This process (including the steps you'll need to take) is documented [here](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3875962884/How+to+submit+an+open+source+contribution+for+Product+Review#Product-Review-Process).
- If it doesn't, simply proceed with the next step.

#### :radio_button: Provide context

To help your reviewers and other members of the community understand the purpose and larger context of your changes, feel free to add as much of the following information to the PR description as you can:

- Dependencies
  > This PR must be merged before / after / at the same time as ...
- Blockers
  > This PR is waiting for OEP-1234 to be accepted.
- Timeline information
  > This PR must be merged by XX date **because** ...
- Partner information
  > This is for a course on edx.org.
- Supporting documentation
- Relevant [Open edX discussion forum](https://discuss.openedx.org/) threads


#### :radio_button: Submit a signed contributor agreement (CLA)

:warning: We ask all contributors to the Open edX project to submit a [signed contributor agreement](https://openedx.org/cla) or indicate their institutional affiliation.
Please see the [CONTRIBUTING](https://github.com/openedx/.github/blob/master/CONTRIBUTING.md) file for more information.

If you've signed an agreement in the past, you may need to re-sign.
See [The New Home of the Open edX Codebase](https://open.edx.org/blog/the-new-home-of-the-open-edx-codebase/) for details.

Once you've signed the CLA, please allow 1 business day for it to be processed.
After this time, you can re-run the CLA check by adding a comment below that you have signed it.
If the CLA check continues to fail, you can tag the `@openedx/cla-problems` team in a comment for further assistance.

<!-- comment:no_cla -->




#### :radio_button: Get a green build

If one or more checks are failing, continue working on your changes until this is no longer the case and your build turns green.



#### :radio_button: Let us know that your PR is ready for review:

### Who will review my changes?


This repository is currently maintained by `@openedx/wg-maintenance-edx-platform`.  Tag them in a comment and let them know that your changes are ready for review.


### Where can I find more information?

If you'd like to get more details on all aspects of the review process for open source pull requests (OSPRs), check out the following resources:

- [Overview of Review Process for Community Contributions](https://docs.openedx.org/en/latest/developers/references/developer_guide/process/FAQ-about-pull-requests.html)
- [Pull Request Status Guide](https://docs.openedx.org/en/latest/developers/references/developer_guide/process/pull-request-statuses.html)
- [Making changes to your pull request](https://docs.openedx.org/en/latest/documentors/how-tos/make_changes_to_your_pull_request.html)

### When can I expect my changes to be merged?

Our goal is to get community contributions seen and reviewed as efficiently as possible.

However, the amount of time that it takes to review and merge a PR can vary significantly based on factors such as:

- The size and impact of the changes that it introduces
- The need for product review
- Maintenance status of the parent repository

:bulb: *As a result it may take up to several weeks or months to complete a review and merge your PR.*

<!-- comment:external_pr -->
<!-- data: eyJkcmFmdCI6IGZhbHNlfQ== -->

mphilbrick211 commented:
Hi @rediris! Thank you for this contribution! Please let us know if you have any questions regarding submitting a [CLA form](https://openedx.atlassian.net/wiki/spaces/COMM/pages/941457737/How+to+start+contributing+to+the+Open+edX+code+base). Thank you!

michaelroytman committed:
feat: add VerificationAttempt model to verify_student application

This commits adds a VerificationAttempt model to store implementation and provider agnostic information about identity verification attempts in the platform.

JadeyOlivier committed:
fix: install dependencies

hamzawaleed01 committed:
feat: Upgrade Python dependency edx-enterprise

feat: added migration for model updateroleassignmentswithcustomersconfig

Commit generated by workflow `openedx/edx-platform/.github/workflows/upgrade-one-python-dependency.yml@refs/heads/master`

pmakwana93 committed:
chore: bumping version num to 4.23.8 (#35310)

JadeyOlivier committed:
feat: updated user retirement docs

JadeyOlivier committed:
fix: dependencies again

JadeyOlivier committed:
fix: change settings config to empty list not dict

navinkarkera committed:
chore: add comment

justinhynes committed:
Revert: revert generic retirement update (#35317)

* Revert "fix: change settings config to empty list not dict"

This reverts commit b65550c79629b572bd2fab783f05f517c02f8f1b.

* Revert "fix: dependencies again"

This reverts commit c06416bed78c8607a83fbe35ea26cc020db70231.

* Revert "feat: updated user retirement docs"

This reverts commit c9641b35d4a4260383c0da04929fbbdf3b8b13b7.

* Revert "fix: install dependencies"

This reverts commit a5442b2409b0e896fd0bb0d2765a5cdcf593d161.

* Revert "Revert "fix: dependencies""

This reverts commit 4cde950007b022718123f54a2685d74e956f527d.

* Revert "fix: dependencies"

This reverts commit 8a1c30ebc517457ce274167d32bdde34440e210f.

* Revert "fix: Add CI update for tests"

This reverts commit 64098b6dabb04f2788840b1e388299bb2a62c943.

* Revert "fix: tests"

This reverts commit 5e636dea169cbb6aeafd6a3f4a9f9f976810f320.

* Revert "fix: generalize internal services"

This reverts commit e8f9db428d6bd41bd17b3ae4bc4ff18fa2fff152.

* Revert "fix: quality"

This reverts commit 77ca0f754af322b918434960f74948665d0847e1.

* Revert "feat: Commerce Coordinator step in retirement pipeline"

This reverts commit c24c87499f75893fbb6b35b2573f361c84a142ab.

yusuf-musleh committed:
feat: REST API to allow pasting clipboard (staged) content into a library (#35199)

eemaanamir committed:
fix: update content reported notification (#35315)

muhammadadeeltajamul committed:
feat: added unsubscribe url in digest header (#35319)

awais786 committed:
feat: Upgrading api to use drf 5th api. register_and_enroll_students api (#35084)

* feat: upgrading simple api to drf compatible.

edx-requirements-bot committed:
chore: Upgrade Python requirements

awais786 committed:
feat: upgrading simple api to drf compatible. (#35260)

kdmccormick committed:
refactor: remove unused builtinblock webpack entry points (#35299)

Most blocks built into edx-platform have a pair of webpack entry
points, like this:

* {BlockName}Display    # js for student+author+public views
* {BlockName}Editor     # js for studio view

Prior to a past build refactoring [1], these entry points were
defined in an auto-genered webpack config file. In order
to simplify the js build process, this config generation
step was removed and the new file webpack.builtinblocks.config.js
was checked directly into edx-platform.

However, during that refactoring, pointless entry
points were introduced for HtmlBlock subclasses
About, CourseInfo, and StaticTab, all of which
just use their superclass's JS.

[1] https://github.com/openedx/edx-platform/issues/32481

farhan committed:
feat: Expose xmodule xblocks Sass variables as vanilla CSS variables (#35233)

* feat: Expose xmodule xblocks Sass variables as vanilla CSS variables
* https://github.com/openedx/edx-platform/issues/35173

kdmccormick reviewed (approved):
Thanks for the fix @rediris . Once you sign the CLA, I'll merge this.

iamsobanjaved committed:
chore: Pin algoliasearch to v3

Recently algoliasearch released the new major version v4 of its Python client and it has multiple breaking changes, which will impact one of the edX platform plugins. So we must make that plugin compatible with algoliasearch v4 before upgrading this package.

Changelog: https://algolia.com/doc/libraries/python/v4/upgrade/

iamsobanjaved committed:
feat: Upgrade Python dependency algoliasearch

Commit generated by workflow `openedx/edx-platform/.github/workflows/upgrade-one-python-dependency.yml@refs/heads/master`

pmakwana93 committed:
chore: bumping version to 4.23.9 (#35335)

Co-authored-by: Kira Miller <31229189+kiram15@users.noreply.github.com>

awais786 committed:
feat: upgrading simple api to drf compatible ( 3rd api ) list_email_content (#35111)

* feat: upgrading simple api to drf compatible.

blarghmatey committed:
fix: Respect the authsource kwarg for MongoDB connections (#35239)

* fix: Respect the authsource kwarg for MongoDB connections

The changes for upgrading to PyMongo 4.4 introduced an authentication
bug in Mongo connections. The `authSource` parameter was being
hard-coded to use the database being connected to. In Mongo the `admin`
db is typically the source of authentication, so unless the user was
explicitly created in the target db then any attempts to connect would
result in authentication failures.

This restores the behavior of allowing for the lowercased `authsource`
kwarg to be used for the `authSource` connection parameter, while
otherwise respecting the operator's configuration parameters.

* fix: Respect the authsource kwarg for MongoDB connections

The changes for upgrading to PyMongo 4.4 introduced an authentication
bug in Mongo connections. The `authSource` parameter was being
hard-coded to use the database being connected to. In Mongo the `admin`
db is typically the source of authentication, so unless the user was
explicitly created in the target db then any attempts to connect would
result in authentication failures.

This restores the behavior of allowing for the lowercased `authsource`
kwarg to be used for the `authSource` connection parameter, while
otherwise respecting the operator's configuration parameters.

nsprenkle committed:
feat: update XBlock to 5.1.0 (#35325)

deborahgu committed:
feat: lint this file (#35348)

I'm about to make a bunch of changes to this file, and before I do I'm
saving it and letting the linter reformatted to our current code style
standards, so that code reviewers won't have to read a mix of lint and
code changes.

FIXES: APER-3554

KristinAoki committed:
Revert "fix: update course discussion config before course load (#35219)" (#35349)

This reverts commit 5c0942481ce292a90f86259bd223d66e7ceffe9f.

e0d commented:
@rediris I notice there are some commit-lint failures.  Please note that we use conventional commits across Open edX projects.  You can read about the details [here](https://open-edx-proposals.readthedocs.io/en/latest/best-practices/oep-0051-bp-conventional-commits.html#type).  Can you please amend your commit messages to follow our standard?

rpenido committed:
feat: add collections app from openedx-learning (#35312)

muhammadadeeltajamul committed:
feat: added snowflake events for email digest unsubscribe (#35329)

AhtishamShahid committed:
feat: added email content in misc notifications (#35341)

mudassir-hafeez committed:
chore: provide logo url from backend for batch enrollment email (#35138)

* chore: provide logo url from backend for batch enrollment email

deborahgu committed:
feat:  linting only (#35370)


Editing this file is going to cause a lot of automatic linting, so lint-only commit for nicer code review.

MueezKhan246 committed:
feat: Upgrade Python dependency edx-enterprise

added encrypted columns for user credentials for SAP config

Commit generated by workflow `openedx/edx-platform/.github/workflows/upgrade-one-python-dependency.yml@refs/heads/master`

awais786 committed:
feat: show_student_extensions upgrading api to drf compatible ( 9th )  (#35148)

* feat: upgrading simple api to drf compatible.

KristinAoki committed:
fix: add aria-current attributes to video captions (#35371)

This PR fixes the accessibility issues associated with only visually treating the current transcript line. The current implementation for the transcript panel bolds the text that is actively being spoken or skipped to. However, there is no aria attribute present to convey this change to assistive technology. This change impacts learners and course authors.

ChrisChV committed:
feat: Problem types added as filterable attribute [FC-0059]  (#35242)

rpenido committed:
feat: allow update xblock fields without passing data (#35320)

navinkarkera committed:
feat: index library collections in studio meilisearch index (#35324)

kdmccormick commented:
@rediris Were you able to sign the CLA? That's the only thing blocking me from merging your contribution.

No need to worry about the committlint failures. Those are happening because of the Merge commits, but when I merge your PR I will squash those all away.

rediris commented:
Hi @kdmccormick I did send a CLA request, but I don't know where I am in the process. I'm going to scrap this PR and start over - this rebasing is turning into too much of a headache.

rediris commented:
Closing to redo the PR.

openedx-webhooks commented:
@rediris Even though your pull request wasn’t merged, please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/edx-platform&entry.867055334=https://github.com/openedx/edx-platform/pull/35301&entry.1484655318=https://github.com/rediris&entry.752974735=2024-08-12+15:28&entry.1917517419=2024-08-27+20:04&entry.2133058324=No) so we can improve your experience in the future.

<!-- comment:end_survey -->
